gitleaks.sh

```diff
@@ -0,0 +1,14 @@
+#script that detects hardcoded secrets and prevents pushing them to repo
```

Comment:
Please locate this file as suggested in instructions

<img width="748" alt="Screen Shot 2020-06-01 at 10 41 10" src="https://user-images.githubusercontent.com/6632944/83386945-8c55c380-a3f4-11ea-8083-7a217ead6378.png">

Reply:
It's a great idea, but for now, I think It's better for us that we can disable it with flags.
cause now we have 96 leaks and we are blocked by this tool until we fix it.